docs/backends/DuckDB.md

```diff
@@ -89,6 +99,10 @@ con = ibis.connect("duckdb://") # (1)
     options:
       heading_level: 4
       show_docstring_returns: false
+::: ibis.backends.duckdb.Backend.read_delta
+    options:
+      heading_level: 4
+      show_docstring_returns: false
```

Comment:
Probably should make this an alphabetical list, too.

Reply:
seems like a larger lift? they don't seem to be in alphabetical order too

what's the `sorted` in `badges.md` suggestion? I'd prefer not to manually sort them 🙃

Reply:
yeah, let's not manually sort the badges.  I was thinking this:

```diff
diff --git a/docs/backends/badges.md b/docs/backends/badges.md
index 8e30aeca6..7201770e7 100644
--- a/docs/backends/badges.md
+++ b/docs/backends/badges.md
@@ -1,3 +1,3 @@
-{% if imports %} ![filebadge](https://img.shields.io/badge/Reads-{{ "%20|%20".join(imports) }}-blue?style=flat-square) {% endif %}
+{% if imports %} ![filebadge](https://img.shields.io/badge/Reads-{{ "%20|%20".join(sorted(imports)) }}-blue?style=flat-square) {% endif %}
 
-{% if exports %} ![exportbadge](https://img.shields.io/badge/Exports-{{ "%20|%20".join(exports) }}-orange?style=flat-square) {% endif %}
+{% if exports %} ![exportbadge](https://img.shields.io/badge/Exports-{{ "%20|%20".join(sorted(exports)) }}-orange?style=flat-square) {% endif %}
```

Reply:
@gforsyth updated w/ the suggestion!